src/services/review/ReviewRequestService.ts

```diff
@@ -234,6 +234,26 @@ export default class ReviewRequestService {
           },
         }))
 
+        // It is a new comment to the user if any of the following
+        // conditions satisfy:
+        // 1. The review request views table does not contain a record
+        //    for the user and the review request.
+        // 2. The review request views table contains a record for that
+        //    user and review request, but the lastViewedAt entry is NULL.
+        // 3. The review request views table contains a record in the
+        //    lastViewedAt entry, and the comment has a timestamp greater
+        //    than the one stored in the database.
+        const allComments = await this.getComments(
+          sessionData,
+          site,
+          pullRequestNumber
+        )
+        const countNewComments = await Promise.all(
+          allComments.map(async (value) => value.isRead)
+        ).then(
+          (arr) => arr.filter((isCommentRead) => isCommentRead === true).length
```

Comment:
sorry to be a pedant but wdyt about this?

```ts
(arr) => {
const readComments = arr.filter(isRead => !!isRead) 
return readComments.length
}
```

Reply:
Ah I think I misunderstood your original intent, adjusted in e315e2a.